### ISSUE TITLE
create stub file not working

### ISSUE THREAD

**Issue by heejaechang** [microsoft/pylance-release#5188]
1. install `pip install zope.event`
2. enable in setting.json
```
"python.analysis.diagnosticSeverityOverrides": {
        "reportMissingTypeStubs": "information"
    },
```
3. create python file and add `import zope.event`
4. place cursor on `event`
5. open `create type stub` code action and run

Expected: type stub created
Actual: Nothing happens.

looks like something around pyright command controller not being called anymore.

**Comment by debonte** [comment]
`PylanceAsyncServer.executeCodeAction` returns (amongst other things) `pyright.createtypestub`. But `BaseCommandController.getCommand` only supports `python.createtypestub`. Looks like there's [history](https://github.com/microsoft/pyrx/pull/209/files#diff-90766567ceac8c3b883630762a14c6c75a0d680ffc913afffae7a8496a9f36f2R25) of us supporting both command names, but it's not clear to me why. This support was broken by https://github.com/microsoft/pyrx/pull/3633.

Reassigning to @rchiodo who should have more context.

**Comment by KacieKK** [comment]
This issue has been fixed in prerelease version 2024.5.100, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#20245100-9-May-2024-PreRelease)
